magit-emacs-Q-command: new command, for debugging

npostavs opened the pull request:
Refer: https://github.com/magit/magit/issues/2856#issuecomment-261773690

> It might be better to show the actual command, possibly prefixed with instructions.

I did that. Not sure if showing the command is a good idea, it's a bit long.

Fixes #2856.

tarsius commented:
>> It might be better to show the actual command, possibly prefixed with instructions.

> I did that. Not sure if showing the command is a good idea, it's a bit long.

On second thought I agree.

Please update all the relevant parts of the documentation.

npostavs commented:
> Please also update all the relevant parts of the documentation.

I updated `ISSUE_TEMPLATE` and added an entry in the release notes, is that what you had in mind?

tarsius commented:
I'v kept the old instructions in the documentation, just in case.
